source/includes/_files.md

```diff
@@ -0,0 +1,199 @@
+# Files
+
+On the OpenDataSoft platform, files can be used as data source to create new datasets.
+
+## The file object
+
+> Example file object
+
+```json
+{
+    "file_id": "cheese_data.csv",
```

Comment:
We talked about it together and we decided that since it doesn't fit the general UID pattern, it should be left as ID. Open to changing it if you changed your mind though.